runtime/themes/naysayer.toml

```diff
@@ -0,0 +1,91 @@
+# Author: Nick Saika <n@nesv.ca>
+#
+# This theme is a port of naysayer-theme.el:
+#    https://github.com/nickav/naysayer-theme.el
+
+"ui.background" = { bg = "bg" }
+"ui.text" = "text"
+"ui.linenr" = { bg = "bg", fg = "line-fg" }
+"ui.linenr.selected" = { bg = "highlight-line", fg = "line-fg" }
+"ui.selection" = { bg = "selection" }
+"ui.cursorline" = { bg = "highlight-line" }
+"ui.statusline" = { fg = "bg", bg = "text" }
+"ui.statusline.inactive" = { fg = "text", bg = "bg" }
+"ui.virtual" = "indent"
```

Comment:
Looks like this is not defined in the palette

Reply:
:facepalm: